data/wanderer/wanderers start.txt

```diff
@@ -989,7 +1065,17 @@ mission "Wanderers: Alpha Surveillance A"
 				`	"Is it possible that they have a hidden base on one of your worlds?"`
 				`	"Do you know where they went when you banned them from your space?"`
 					goto where
+				`	"I am pretty sure I found one of their old bases."`
+					to display
+						has "found alpha base"
+					goto oldbase
 			`	"No," he says, "It has been two hundred years, and our worlds are populous and overseen with great diligence. Unless they have taken refuge with the Unfettered, they are not in our space."`
+			`	"Then where could they have gone?" you ask.`
+				goto where
+			label oldbase
+			action
+				set "told them the alpha base location"
+			`	"An old base, you say? They probably no longer operate there anymore then. I would advise against going there again, because it most likely will be trapped."`
```

Comment:
```suggestion
			`	"An old base, you say? They probably no longer operate there anymore then. I would advise against going there again, because it will most likely have traps."`
```